src/remote-config/remote-config.ts

```diff
@@ -254,6 +282,143 @@ class RemoteConfigTemplateImpl implements RemoteConfigTemplate {
   }
 }
 
+/**
+ * Remote Config data-plane template data implementation.
+ */
+class RemoteConfigServerTemplateImpl implements RemoteConfigServerTemplate {
+  public cache: RemoteConfigServerTemplateData;
+
+  constructor(
+    private readonly apiClient: RemoteConfigApiClient,
+    public readonly defaultConfig: RemoteConfigServerConfig = {}
+  ) { }
+
+  /**
+   * Fetches and caches the current active version of the {@link RemoteConfigServerTemplate} of the project.
+   */
+  public load(): Promise<void> {
+    return this.apiClient.getServerTemplate()
+      .then((template) => {
+        this.cache = new RemoteConfigServerTemplateDataImpl(template);
+      });
+  }
+
+  /**
+   * Evaluates the current template in cache to produce a {@link RemoteConfigServerConfig}
+   */
+  public evaluate(): RemoteConfigServerConfig {
+    if (!this.cache) {
+      throw new FirebaseRemoteConfigError(
+        'failed-precondition',
+        'No Remote Config Server template in cache. Call load() before calling evaluate().');
+    }
+
+    const evaluatedConfig: RemoteConfigServerConfig = {};
+
+    for (const [key, parameter] of Object.entries(this.cache.parameters)) {
+      const { defaultValue, valueType } = parameter;
+
+      if (!defaultValue) {
+        console.debug(`Filtering out parameter ${key} with no default value`);
```

Comment:
@lahirumaramba our client SDKs log messages in cases like this, but I don't see other usages of `console` in the Admin SDK. Any objections to adding logging like this?

Reply:
Scratch that. I see several usages of `console.log`. I must have been looking for `console.debug` before. I've updated this to use `console.log`, for consistency.

Reply:
Removed, per discussion.